bundles/specmate-auth/src/com/specmate/auth/internal/UserSession.java

```diff
@@ -0,0 +1,29 @@
+package com.specmate.auth.internal;
+
+import java.util.Date;
+
+public class UserSession {
+	private static final long MAXIMUM_AGE = 60 * 60 * 12 * 1000L; // 12 hours in milliseconds
```

Comment:
Im future, we might want to make this configurable via a config-file. I think we should make this configurable right away.

Reply:
Ok, will fix.